src/instructlab/configuration.py

```diff
@@ -1290,6 +1290,7 @@ def map_train_to_library(ctx, params):
 
     train_args.deepspeed_options = ds_args
     train_args.lora = lora_args
+    train_args.disable_flash_attn = True
```

Comment:
Why the change here?

Reply:
good catch. This really needs to be a flag bc when training with granite w/out cuda, this fails

Reply:
actually for now I am just going to do this to avoid cluttering the flags even more

```
    if params["pipeline"] == "full":
        train_args.disable_flash_attn = True
```